bin/commands/defrag.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+module.exports = {
+  command: 'defrag',
+  describe: 'synchronize all dependency version discrepancies',
+  builder: {
```

Comment:
this is very exciting :D 

these options provide much more utility than my one-off script does